include/ocpp/v201/device_model.hpp

```diff
@@ -91,13 +91,13 @@ typedef std::function<void(const VariableMonitoringMeta& updated_monitor, const
                            const VariableAttribute& attribute, const std::string& current_value)>
     on_monitor_updated;
 
-/// \brief This class manages access to the device model representation and to the device model storage and provides
+/// \brief This class manages access to the device model representation and to the device model interface and provides
 /// functionality to support the use cases defined in the functional block Provisioning
 class DeviceModel {
 
 private:
     DeviceModelMap device_model;
-    std::unique_ptr<DeviceModelStorage> storage;
+    std::unique_ptr<DeviceModelInterface> interface;
```

Comment:
nit: I would rather call this `device_model_interface` or just `device_model` and rename the 
DeviceModelMap device_model to `device_model_map`